docs/guides/ui.md

```diff
@@ -0,0 +1,278 @@
+2. History navigation returns to previous views, similar to the back button in a web browser.
+3. Add new tab opens a new code editor window.
+4. Plan opens the plan module.
+5. Documentation links to the SQLMesh documentation website.
+6. The crescent moon toggles between page light and dark modes.
+7. Run SQL query executes the [`sqlmesh fetchdf` command](../reference/cli.md#fetchdf).
+8. Format SQL query reformats a SQL query using SQLGlot's pretty layout.
+9. Change SQL dialect specifies the SQL dialect of the current tab for custom SQL queries. It does not affect the SQL dialect for the project.
+
+![SQLMesh browser UI buttons](./ui/ui-guide_ui-startup-buttons.png)
+
+And it contains four status indicators:
+
+1. Editor tab language displays the programming language of the current code editor tab (SQL or Python).
+2. Current environment displays the currently selected environment
+3. Change indicator displays a summary of the changes in the project files relative to the most recently run SQLMesh plan in the selected environment.
+4. Error indicator displays the count of errors in the project.
+
+![SQLMesh web UI status indicators](./ui/ui-quickstart_ui-startup-status.png)
+
+#### Edit models
+
+Open a model in a new tab by clicking its file name in the left-hand project directory pane.
+
+The tab will show the model definition, and the details pane at the bottom will display the model in the project's table and column lineage.
+
+![Incremental model open in editor](./ui/ui-quickstart_incremental-model.png)
+
+The lineage display will update as model modifications are saved. For example, you might modify the incremental SQL model by adding a new column to the query. Press `Cmd + S` (`Ctrl + S` on Windows) to save the modified model file and display the updated lineage:
+
+![Incremental model modified in editor](./ui/ui-quickstart_incremental-model-modified.png)
+
+The `Changes` indicator in the top right now shows blue and orange circles that reflect our model update.
+
+#### Run SQL queries
+
+Run SQL queries by executing them from custom SQL editor tabs.
+
+For example, we might add a SQL query `select * from sqlmesh_example.incremental_model` to the Custom SQL 1 tab. To run the query, first click the hamburger icon to open the explorer pane:
+
+![Querying `dev` incremental model with SQL query in editor](./ui/ui-guide_fetchdf-prod-query.png)
+
+Then click the `Run Query` button in the bottom right to execute the query:
+
+![Results from querying dev incremental model with SQL query in editor](./ui/ui-guide_fetchdf-prod.png)
+
+The results appear in an interactive table in the details pane below the editor.
+
+### Plan module
+
+The plan module provides a graphical interface for the `sqlmesh plan` command. Its actions change based on the current state of your SQLMesh project.
```

Comment:
```suggestion
The plan module provides a graphical interface for the `sqlmesh plan` command. Its actions change based on the current state of your SQLMesh project. For example, it will auto-update based on the latest saved changes to a SQL model.
```